lib/rucio/daemons/oauthmanager/oauthmanager.py

```diff
@@ -38,140 +38,139 @@
 
 import rucio.db.sqla.util
 from rucio.common.exception import DatabaseException
-from rucio.common.logging import setup_logging, formatted_logger
-from rucio.common.utils import daemon_sleep
+from rucio.common.logging import setup_logging
 from rucio.core.authentication import delete_expired_tokens
-from rucio.core.heartbeat import die, live, sanity_check
+from rucio.core.heartbeat import sanity_check
 from rucio.core.monitor import record_counter, record_timer
 from rucio.core.oidc import delete_expired_oauthrequests, refresh_jwt_tokens
+from rucio.daemons.common import run_daemon
 
-GRACEFUL_STOP = threading.Event()
+graceful_stop = threading.Event()
 
 
-def OAuthManager(once=False, loop_rate=300, max_rows=100, sleep_time=300):
+def OAuthManager(once: bool = False, max_rows: int = 100, sleep_time: int = 300) -> None:
     """
     Main loop to delete all expired tokens, refresh tokens eligible
     for refresh and delete all expired OAuth session parameters.
     It was decided to have only 1 daemon for all 3 of these cleanup activities.
 
     :param once: If True, the loop is run just once, otherwise the daemon continues looping until stopped.
-    :param loop_rate: obsolete, please use sleep_time instead. The number of seconds the daemon will wait before running next loop of operations.
     :param max_rows: Max number of DB rows to deal with per operation.
     :param sleep_time: The number of seconds the daemon will wait before running next loop of operations.
 
     :returns: None
     """
-    if sleep_time == OAuthManager.__defaults__[3] and loop_rate != OAuthManager.__defaults__[1]:
-        sleep_time = loop_rate
 
-    executable = 'oauth-manager'
+    run_daemon(
+        once=once,
+        graceful_stop=graceful_stop,
+        executable='oauth-manager',
+        logger_prefix='OAuthManager',
```

Comment:
unless there is a particular reason to have it different, I would just set `executable` = `logger_prefix` = 'oauth-manager'. Especially, I see that the logger prefix is `oauth_manager` in the code, so I don't understand why 'OAuthManager' here?

Reply:
No reason, just kept the convention that was already in place, will change.